documentation/modules/exploit/multi/fileformat/libreoffice_logo_exec.md

```diff
@@ -0,0 +1,90 @@
+## Description
+
+  This module exploits CVE-2019-9848 and is based on the module exploiting CVE-2018-16858, written by Shelby Pace.
+
+  LibreOffice has a feature where documents can specify that pre-installed scripts can be executed on various document events such as mouse-over, etc. LibreOffice is typically also bundled with LibreLogo, a programmable turtle vector graphics script, which can be manipulated into executing arbitrary python commands. 
+
+  By using the document event feature to trigger LibreLogo to execute python contained within a document a malicious document could be constructed which would execute arbitrary python commands silently without warning. 
+
+  This module generates an ODT file with a dom loaded event that, when triggered, will execute any arbitrary python code and the metasploit payload. LibreLogo executes the python code stored on the text part of the document.
+
+  The generated document file contains a one-liner python code that does an os.system : 
+
+  ```python
+  getattr(__import__(“os”),“system”)(“<%= @cmd %>”)
+  ```
+
+  but encoded :
+
+  ```xml
+  <text:p text:style-name="P8">&#x67;&#x65;&#x74;&#x61;&#x74;&#x74;&#x72;(&#x5f;&#x5f;&#x69;&#x6d;&#x70;&#x6f;&#x72;&#x74;&#x5f;&#x5f;(&#x201C;\x6f\&#x78;73&#x201D;),&#x201C;\&#x78;73\&#x78;79\&#x78;73\&#x78;74\x65\&#x78;6d&#x201D;)(“<%= @cmd %>”)</text:p>
+  ```
+
+  To avoid any python error, the h1 title written in the document is a python comment `#`.
+
+## Vulnerable Application
+
+  "LibreOffice versions prior to 6.2.5", but, unfortunately, it seems to also work on 6.2.5.
```

Comment:
```suggestion
  LibreOffice version 6.2.5 and prior.
```

Also, list the versions of LibreOffice with which the module was tested here. ie:

```
This module has been tested successfully with:

* LibreOffice 6.2.4 on Windows 7
* LibreOffice 6.2.4 on Debian 9.9
```